db/controllers/controllers.js

```diff
@@ -26,51 +27,67 @@ next(err);
 };
 
 exports.getAllArticles = (req, res, next) => {
-selectAllArticles().then((articles) => {
-  res.status(200).send({ articles });
-})
-.catch((err) => {
-  next(err);
-});
-};
+    selectAllArticles().then((articles) => {
+        res.status(200).send({ articles });
+      })
+      .catch((err) => {
+        next(err);
+      });
+  };
 
 
-exports.getComments = (req, res, next) => {
-const { article_id } = req.params;
-selectComments(article_id)
-.then((comments) => {
-  res.status(200).send({ comments });
-})
-.catch(next);
-};
+  exports.getComments = (req, res, next) => {
+    const { article_id } = req.params;
+    selectComments(article_id)
+      .then((comments) => {
+        res.status(200).send({ comments });
+      })
+      .catch(next);
+  };
+  
+  exports.postComment = (req, res, next) => {
 
-exports.postComment = (req, res, next) => {
+    const body = req.body
+    const article_id = req.params.article_id
 
-const body = req.body
-const article_id = req.params.article_id
+    checkArticleId(article_id)
+    .then(() => {
+      return insertComment(body, article_id)
+    })
+    .then((comment) => {
+      res.status(201).send(comment);
+    })
+    .catch(next);
+  };
 
-checkArticleId(article_id)
-.then(() => {
-return insertComment(body, article_id)
-})
-.then((comment) => {
-res.status(201).send(comment);
-})
-.catch(next);
-};
+  exports.patchArticleById = (req, res, next) => {
+    const { article_id } = req.params
+    const { inc_votes } = req.body
 
-exports.patchArticleById = (req, res, next) => {
-const { article_id } = req.params
-const { inc_votes } = req.body
+    checkArticleId(article_id)
+    .then(() => {
+       return updateArticleById(article_id, inc_votes)
+     })
 
-checkArticleId(article_id)
-.then(() => {
- return updateArticleById(article_id, inc_votes)
-})
+    .then((article) => {
+        res.status(201).send({ article })
+    })
+    .catch(next)
+  }
+  
+
+
+  exports.deleteCommentById = (req, res, next) => {
+    const { comment_id } = req.params;
+
+    checkCommentId(comment_id)
```

Comment:
We don't need to check this first, you can add a custom error in your model that is handling the id not found error.